components/site/momentumNavigation.tsx

```diff
@@ -15,101 +23,126 @@ export interface MomentumNavigationItemProps {
   level?: number;
 }
 
-export type MomentumNavigationProps = {
-  data?: MomentumNavigationItemProps[];
-};
-
 const StyledLink = styled(Box)<BoxProps & { $active?: boolean }>`
   cursor: pointer;
   overflow-wrap: anywhere;
-  a, a:visited, a:hover, a:active {
+  a,
+  a:visited,
+  a:hover,
+  a:active {
     color: inherit;
     text-decoration: none;
   }
   &:hover {
     ${({ $active }) => {
       return css({
         bg: $active ? 'blue.700' : 'gray.200',
-        color: $active ? 'white' : 'inherit'
-      })
+        color: $active ? 'white' : 'inherit',
+      });
     }}
   }
   ${({ $active }) => {
     return css({
       bg: $active ? 'blue.700' : 'transparent',
-      color: $active ? 'white' : 'inherit'
-    })
+      color: $active ? 'white' : 'inherit',
+    });
   }}
 `;
 
 const StatusColorMap = {
   none: 'green.700',
   minor: 'yellow.400',
   major: 'brand.orange',
-  critical: 'red.700'
-}
+  critical: 'red.700',
+};
 
-const MomentumNavigation = (props: MomentumNavigationProps): JSX.Element | null => {
+const MomentumNavigation = (): JSX.Element | null => {
   const { status } = useStatus();
   const environment = getWindow();
-  
-  if (!props.data) {
-    return null;
-  }
-
 
   return (
     <Box width="260px" position="sticky" top="0">
       <StyledLink $active={environment?.location?.pathname === '/momentum'}>
         <Link href="/momentum" passHref>
-          <Box as="a" display="inline-block" fontSize="200" fontWeight="semibold" py="200" lineHeight="200" px="500">Momentum Documentation</Box>
+          <Box
+            as="a"
```

Comment:
Idk why there are prettier changes here, do we need to adjust our setup?

Reply:
Not sure why either, maybe I need to check my local setup